builder/bin/publish-release-controller.go

```diff
@@ -77,15 +78,12 @@ func main() {
 
 	res, err := client.Do(req)
 
-	if res.StatusCode == 404 {
-		fmt.Println("Check the Controller. Is it running?")
-		os.Exit(1)
+	if err != nil {
+		log.Fatalln(err)
 	}
 
-	if err != nil || res.StatusCode != 200 {
-		fmt.Println("failed retrieving config from controller")
-		fmt.Println(res.Body)
-		os.Exit(1)
+	if res.StatusCode != 200 {
+		log.Fatalf("failed retrieving config from controller: %s\n", res.Body)
```

Comment:
The thing that confuses me with the check for a 404 response is that the router [returns a 503](https://github.com/deis/deis/blob/master/router/image/templates/nginx.conf#L83-L85) if the controller is down. The only time we'd ever respond with a 404 for a push hook is if [the user or application is not found](https://github.com/deis/deis/blob/master/controller/api/views.py#L270-L271), which means that the error message was misleading. Should I change that to a 503 check instead?

ping @aledbf in case he knows more about this specific check